### PR TITLE
Adds hero image and about me place holder

### PR DESCRIPTION
**Pull request by hailed748**
First real pull request towards the week 1 project. Added a hero image and setup the about me portion with some dummy text.

Question: 
If you look at the file I attached you can see that the middle text is not quite center relative to the image. I tried for a while but could not get the results I wanted. Do you have any suggestion on a fix? 

Screenshot 2020-06-19 at 3 49 09 PM

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,9 +1,39 @@
...
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body,
+html {
+    margin: 0;
+    padding: 0;
+}
+.jumbotron {
+    align-content: center;
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/style.css:7]: Try align-items instead, I think that might be what you're looking for

Comment (hailed748) [reply portfolio/src/main/webapp/style.css:7]: That was it, along with a small margin change. How would I update this commit with this change, or do I just make a new one?

Comment (LukeGMTravis) [reply portfolio/src/main/webapp/style.css:7]: Updating the branch should update the PR accordingly I think

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,63 @@
...
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="jumbotron">
+        <h1 class="headline-text"> Dagmawi Haile </h1>
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/index.html:11]: Remove whitespace padding

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,79 @@
...
+    align-items: center;
+    display: flex;
+    background-image: linear-gradient(rgba(0,0,0,0.7),rgba(0,0,0,0.7)),url("/images/jumboBack.jpg");
+    background-position: center center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 50vh;
+    justify-content: center;
+    position: relative;
+}
```
Comment (Tetheta) [inline portfolio/src/main/webapp/style.css:16]: FYI typically we sort css properties alphabetically (many IDEs have a sort function that can do it for you).
Makes things less confusing to refactor later

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -1,9 +1,79 @@
...
+
+#about-title {
+    font-family: 32px Arial, Helvetica, sans-serif;
+    font-weight: bold;
+    text-align: center;
+    text-decoration: underline;
+}
+.about {
+    font-family: 20px/3 Arial, Helvetica, sans-serif;
```
Comment (Tetheta) [inline portfolio/src/main/webapp/style.css:39]: I haven't seen the 20px/3 in a font family before. What's the purpose there?

Comment (hailed748) [reply portfolio/src/main/webapp/style.css:39]: It was supposed to be "font" my bad. I saw it in the style guide, It's a shorthand to combine font-family, font-size, and line-height 
Screenshot 2020-06-22 at 11 32 47 AM

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (hailed748) [reply portfolio/src/main/webapp/style.css:16]: Okay I was doing this, might have missed a few cases like "display". Does this apply to the entire CSS file as well, should the classes/id be in alphabetical order?

Comment (LukeGMTravis) [reply portfolio/src/main/webapp/style.css]: Just the individual attributes should be in alphabetical order. Classes/ids/etc can be however makes the best logical sense.

File: portfolio/src/main/webapp/style.css (hunk 4 above)
Comment (Tetheta) [reply portfolio/src/main/webapp/style.css]: Ah that makes more sense.

**Pull request merged**